docs/dml-of-carbondata.md

```diff
@@ -529,6 +529,10 @@ CarbonData DML statements are documented here,which includes:
   * Level 1: Merging of the segments which are not yet compacted.
   * Level 2: Merging of the compacted segments again to form a larger segment.
 
+  The segment whose data size exceed limit of carbon.minor.compaction.size will not be included in
+  minor compaction. If user want to control the size of segment included in minor compaction,
+  configure the property with appropriate value in MB, if not configure, will merge segments only
+  based on number of segments.
```

Comment:
change to below

`User can control the size of a segment to be included in the minor compaction by using carbon.minor.compaction.size. If not configured, minor compaction will consider the segments based on carbon.compaction.level.threshold by neglecting the size of each segment`